roles/tendrl-storage-node/tasks/tendrl-node-agent.yml

```diff
@@ -22,6 +22,8 @@
     - restart tendrl-node-agent
   when: tendrl_gluster_provisioning_support == True
   run_once: true
+  notify:
```

Comment:
Have you tested this?
By looking at the code, this block gets executed only if the tendrl_gluster_provisioning_support == True; which is not supported now